packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@sphereon/ssi-sdk-did-auth-siop-authenticator",
+  "version": "0.0.1",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IDidAuthSiopOpAuthenticator": "./src/types/IDidAuthSiopOpAuthenticator.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "generate-plugin-schema": "yarn veramo dev generate-plugin-schema"
+  },
+  "dependencies": {
+    "@sphereon/did-auth-siop": "^0.2.2",
+    "@sphereon/pe-js": "^0.5.0",
+    "@veramo/core": "^3.1.0",
```

Comment:
Although probably not problematic, makes sense to use the same version as the other @veramo packages, given the whole SSI-SDK will need to use the same versions of upstream plugins

Reply:
i do not see any differences, we use the same version everywhere

Reply:
"@veramo/utils": "^3.1.1-next.17",

Reply:
there is no @veramo/utils 3.1.0, seems they are still working on their initial release